spline-shadow/client-ui/src/app/components/lineage/lineage-details/schema-details/schema-details.component.ts

```diff
@@ -31,10 +32,12 @@ export class SchemaDetailsComponent implements AfterViewInit {
 
   public detailsInfo: any = null
```

Comment:
Maybe there could be some interface that could limit what properties ```details``` could have.

Reply:
yes indeed but I will modify the type when the rest api will be ready and the types defined and exported via swagger. From now, we have nothing defined yet.